tests/dbTestCase.php

```diff
@@ -16,8 +16,8 @@ protected function setUp()
     $conf = parse_ini_file(__DIR__.'/config.ini');
 
     // Make sure we have some clean, temporary output dir
-    exec( "rm -rf " . escapeshellarg($conf['savedir']) );
-    @mkdir( $conf['savedir'], 0777, TRUE );
+    @rmdir($conf['savedir']);
```

Comment:
rmdir will fail to remove the directory if it's not empty, that's why it was rm -rf before